src/Microsoft.OData.Core/UriParser/SyntacticAst/ExpandTermToken.cs

```diff
@@ -176,121 +116,41 @@ public ExpandTermToken(PathSegmentToken pathToNavigationProp, QueryToken filterO
             ExpandToken expandOption,
             ComputeToken computeOption,
             IEnumerable<QueryToken> applyOptions)
+            : base(pathToNavigationProp, filterOption, orderByOptions, topOption, skipOption, countQueryOption, searchOption, selectOption, expandOption)
         {
-            ExceptionUtils.CheckArgumentNotNull(pathToNavigationProp, "property");
-
-            this.pathToNavigationProp = pathToNavigationProp;
-            this.filterOption = filterOption;
-            this.orderByOptions = orderByOptions;
-            this.topOption = topOption;
-            this.skipOption = skipOption;
-            this.countQueryOption = countQueryOption;
-            this.levelsOption = levelsOption;
-            this.searchOption = searchOption;
-            this.selectOption = selectOption;
-            this.computeOption = computeOption;
-            this.expandOption = expandOption;
-            this.applyOptions = applyOptions;
+            LevelsOption = levelsOption;
+            ComputeOption = computeOption;
+            ApplyOptions = applyOptions;

```

Comment:
@mike, does select support $compute, $apply from OData spec future?

Reply:
Updated: 6/18/2019: 
$compute should be supported as a valid select option. It was there in the ABNF and URL Conventions, but was missing from the Protocol spec